api/floor/schema.py

```diff
@@ -109,6 +110,14 @@ def resolve_get_rooms_in_a_floor(self, info, floor_id):
         rooms = query.filter(RoomModel.floor_id == floor_id)
         return rooms
 
+    @Auth.user_roles('Admin')
+    def resolve_filter_by_block(self, info, blockId):
+        query = Floor.get_query(info)
+        floors = query.filter_by(block_id=blockId)
+        if not floors:
```

Comment:
Also, write a test for this case

Reply:
@Bonifase I have implemented the requested change.

Reply:
Use Andela engineering-playbook as a guide for your PR description

Reply:
Sure, I will get on to that immidietly.